internals/secrethub/credential_backup_test.go

```diff
@@ -0,0 +1,128 @@
+package secrethub
+
+import (
+	"bytes"
+	"errors"
+
+	"github.com/secrethub/secrethub-go/internals/assert"
+
+	"strings"
+
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+
+	"testing"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+)
+
+func TestCredentialBackupCommand_Run(t *testing.T) {
+	var testError = errors.New("test")
+
+	testCases := map[string]struct {
+		cmd            CredentialBackupCommand
+		promptOut      string
+		out            string
+		in             string
+		username       string
```

Comment:
This struct field seems to be unused, so it should be removed.

Reply:
Nice spot! Thanks. :D